app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/DRecordDetailsFragment.kt

```diff
@@ -74,8 +81,27 @@ class DRecordDetailsFragment : Fragment() {
             putBoolean("isSubFragment", true)
         }
 
+        var currentContact: Contact? = null
```

Comment:
Just a last remark here, consider using contactsViewModel.contacts.value?.let {
 // and write the logic here with a non nullable `currentContact`
}